app/assets/stylesheets/application.scss

```diff
@@ -165,7 +190,7 @@
  }
 
  .post-it-placeholder {
- background: $dark-grey;
+ background: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.